eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -109,6 +118,13 @@ jobs:
               artifactName: 'artifacts' 
               targetPath: $(Build.ArtifactStagingDirectory)
 
+          - task: PythonScript@0
+            condition: eq(variables['SetDevVersion'],'true')
```

Comment:
Condition should include `SucceededorFailed()`?

Reply:
Discussed with Praveen, this is deliberately identical to the same steps in the JS repo.